src/client/src/apps/SpotlightRoute/handleIntent.ts

```diff
@@ -29,10 +29,10 @@ function showCurrencyPair(currencyPair: string, { window }: PlatformAdapter) {
 }
 
 export const handleIntent = (
-  response: DetectIntentResponse[],
+  response: DetectIntentResponse,
   platformAdapter: PlatformAdapter,
 ) => {
-  const queryResult = response[0].queryResult
+  const queryResult = response.queryResult
```

Comment:
why this has changed?

Reply:
is it not an array anymore?

Reply:
i see now, ignore please

Reply:
I guess I can leave it as it was. I'm just not sure we ever get more than one result